lib/screens/assessment.dart

```diff
@@ -0,0 +1,166 @@
+                });
+              },
+              itemBuilder: (context, index) {
+                return SingleChildScrollView(
+                  child: Column(
+                    children: [
+                      assessments[index],
+                      // Container(
+                      //   width: MediaQuery.of(context).size.width,
+                      //   height: 10,
+                      //   color: Theme.of(context).cardColor,
+                      //   child: ListView.builder(
+                      //     scrollDirection: Axis.horizontal,
+                      //     itemCount: assessments.length,
+                      //     itemBuilder: (context, i) {
+                      //       return Padding(
+                      //         padding: const EdgeInsets.all(2.0),
+                      //         child: Container(
+                      //           width: index == i ? 40 : 20,
+                      //           color: Colors.white70,
+                      //         ),
+                      //       );
+                      //     },
+                      //   ),
+                      // ),
+                    ],
+                  ),
+                );
+              },
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(
+              15.0,
+              5.0,
+              15.0,
+              15.0,
+            ),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Container(
+                  child: Row(
+                    children: List.generate(
+                      assessments.length,
+                      (index) => buildDot(index, context),
+                    ),
+                  ),
+                ),
+                SizedBox(
+                  height: 30,
+                  child: TextButton(
+                    onPressed: () {
+                      _controller.nextPage(
+                        duration: Duration(milliseconds: 100),
+                        curve: Curves.bounceIn,
+                      );
+                    },
+                    child: Text(
+                      currentIndex == assessments.length - 1
+                          ? 'Submit'
+                          : 'Next',
+                      style: TextStyle(color: Colors.white),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  GestureDetector buildDot(int index, BuildContext context) {
+    return GestureDetector(
+      onTap: () {
+        _controller.jumpToPage(index);
+      },
+      child: Container(
+        margin: EdgeInsets.all(2.0),
+        height: 10,
+        // width: 10,
+        width: currentIndex == index ? 20 : 10,
+        decoration: BoxDecoration(
+          color: currentIndex == index ? Colors.orange : Colors.white70,
```

Comment:
instead of orange, I think changing it to like "Color.fromARGB(230, 44, 88, 82)" would fit the color scheme of the app more